src/components/Cards/Card.jsx

```diff
@@ -0,0 +1,52 @@
+import './Card.css';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+
+function Card({ name }) {
+    const [poki, setPoki] = useState([]);
+
+    useEffect(() => {
+        async function fetchPokemons() {
+            try {
+                const result = await axios.get(`https://pokeapi.co/api/v2/pokemon/${name}/`);
+                setPoki(result.data);
+                console.log('card data', result.data);
+                // console.log('abilities', result.data.abilities)
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+
+        fetchPokemons();
+    }, []);
+
```

Comment:
Wat je ook nog kan doen is hier al returnen als `poki` undefined is
```suggestion
        if (!poki) return null;
```

Dan hoef je verderop geen checks meer te doen